include/coreinit/interrupts.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include <wut.h>
+
+/**
+ * \defgroup coreinit_interrupts Interrupt Management
+ * \ingroup coreinit
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum OSInterruptType
+{
+   INTERRUPT_TYPE_ERROR         = 0,
```

Comment:
I think we prefix all the OS* enums with OS_, so OS_INTERRUPT_TYPE_ERROR.

Reply:
Done